pymc4/util/interceptors.py

```diff
@@ -9,7 +9,7 @@
     'CollectLogProb'
 ]
 
-VariableDescription = collections.namedtuple('VariableDescription', 'Dist,shape')
+VariableDescription = collections.namedtuple('VariableDescription', 'Dist,shape,rv')
```

Comment:
I'm still worried about this solution. Variable description is supposed to be collected far before sampling (or what about changing this?). So when you first collect VariableInfo for the first time and get these RVs, you save temporary nodes. When you collect LogProb you again run the model and variables involves there are totally different from those that are stored in `VariableDescription`. That's why I did not store them there.

Reply:
The problem i am facing if RVs are not stored in the VariableDescription is that it doesn't store the specifics of any distribution (like loc or scale) even if they are mentioned in the model definition. So we will have to collect all this already provided info somehow.

``` python
@model.define
def process():
    mu = ed.Normal(loc=0., scale=10., name="mu")
    # here we lose the info that it has loc 0 and scale 10 without RV.
```

We can try defining a new Interceptor which does this for us for each RV.
We can then overwrite(replace existing and add new) the collected data every-time we call configure.

Reply:
OK, let's make it later